R/DatabaseConnections.R

```diff
@@ -29,7 +29,7 @@ ConnectionHandler <- R6::R6Class(
     },
 
     renderTranslateSql = function(query, ...) {
-      sql <- SqlRender::render(sql = query, ...)
+      sql <- SqlRender::render(sql = query, limit_row_count = Sys.getenv("LIMIT_ROW_COUNT"), ...)
```

Comment:
How about this instead of modifying every query?

```

if(limitRowCount) { # Check for not null and positive integer
    sql <- SqlRender::render("SELECT TOP @limit_row_count FROM (@sub_query) results;", limit_row_count = Sys.getenv("LIMIT_ROW_COUNT", sub_query = sql)
}
```

I don't think there are any cases in the code where this would have a bad consequence but an overide flag could be added to the Backend class.

Reply:
it's probably better to add this to the queryDb function rather than the renderTranslate call